src/main/java/pageObjects/Admin_Portal/AdminPortalTest1stPage.java

```diff
@@ -0,0 +1,281 @@
+    private WebElement Statuses;
+
+    public void clickStatus() {
+        hoverOverElementUsingJS(Title);
+        Title.click();
+        Statuses.click();
+
+    }
+
+    @FindBy(linkText = "Priorities")
+    private WebElement Priorities;
+
+    public void clickPriorities() throws InterruptedException {
+        sleep();
+        hoverOverElementUsingJS(Title);
+        Title.click();
+        hoverOverElementUsingJS(Priorities);
+        Priorities.click();
+
+    }
+
+
+    @FindBy(xpath = "//button[@class='hf-mod-create']")
+    private WebElement NewStatus;
+
+    public void clickNewStatus() {
+        NewStatus.click();
+    }
+
+
+    @FindBy(xpath = "//input[@aria-label='Status Name']")
+    private WebElement StatusName;
+
+    public void enterStatusName(String text) {
+        StatusName.sendKeys(text);
+    }
+
+
+    @FindBy(xpath = "//div[@class='sp-preview-inner']")
+    private WebElement StatusColourInner;
+
+    public void clickStatusColourInner() {
+        StatusColourInner.click();
+    }
+
+
+    @FindBy(xpath = "//input[@placeholder='Enter any valid color code format.']")
+    private WebElement StatusColour;
+
+    public void enterStatusColour(String text) {
+        clickStatusColourInner();
+        StatusColour.clear();
+        StatusColour.sendKeys(text);
+        clickStatusColourInner();
+    }
+
+
+    @FindBy(xpath = "//div[@aria-label='Behavior']")
+    private WebElement Behavior;
+
+    public void enterBehavior(String text) {
+        Behavior.click();
+        Behavior.sendKeys(text);
+        Behavior.sendKeys(Keys.ENTER);
+
+    }
+
+
+    @FindBy(xpath = "//textarea[@aria-label='Description']")
+    private WebElement StatusDescription;
+
+    public void enterStatusDescription(String text) {
+        StatusDescription.clear();
+        StatusDescription.sendKeys(text);
+    }
+
+
+    @FindBy(xpath = "//button[@data-test-id='add-status']")
+    private WebElement AddStatus;
+
+    public void clickAddStatus() {
+        AddStatus.click();
+    }
+
+    @FindBy(xpath = "//a[@data-test-id='manage-statuses-left-nav']")
+    private WebElement StatusesSection;
+
+    public void clickStatusesSection() throws InterruptedException {
+        scrolltoElement(StatusesSection);
+        StatusesSection.click();
+    }
+
+
+    public void setDefaultStatus(String xpath) throws InterruptedException {
+        String Status = "//div[contains(text(),'" + xpath + "')]//following::td[3]";
+        WebElement abc = driver.findElement(By.xpath(Status));
+        hoverOverElementUsingJS(abc);
+        abc.click();
+    }
+
+
+    @FindBy(xpath = "//a[contains(text(),'Priorities')]")
+    private WebElement PrioritySection;
+
+    public void clickPrioritySection() {
+        PrioritySection.click();
+    }
+
+    @FindBy(xpath = "/html[1]/body[1]/div[3]/div[1]/section[1]/section[1]/div[1]/div[1]/section[1]/div[1]/div[1]/div[2]/div[1]/div[2]/div[1]/table[1]/tbody[1]/tr[9]/td[2]")
```

Comment:
Usage of absolute Xpath is always not recommended. If DOM changes, then this locaiton won't work. 
It is better use other locators such as id,name, relative xpath etc.